docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -212,10 +212,44 @@ \section{Safety and Security Requirements}
 \wss{Newly discovered requirements.  These should also be added to the SRS.  (A
 rationale design process how and why to fake it.)}
 
```

Comment:
Maybe include a sentence outlining where to find our other requirements and a link to the srs for clarity